src/frontend/package.json

```diff
@@ -1,48 +1,34 @@
 {
-  "name": "@seneca/telescope-frontend",
+  "name": "telescope-frontend",
+  "version": "3.0.0",
   "private": true,
+  "scripts": {
+    "develop": "next dev -p 8000",
+    "build": "next build",
+    "start": "next start",
+    "eslint": "eslint pages -c .eslintrc.json --ext js,jsx,ts,tsx",
```

Comment:
Similar question here: how does this play with our top-level eslint script?  Should we hook this up to the top-level eslint script as well?  Do we need to ignore anything in order to let this happen in the frontend differently?  Can eslint deal with different configs at different levels int he same run?

Reply:
To be honest, I'm not sure how it's going to work. My first idea was to run all the scripts from `src/frontend` and use `dev` or `prod` as backend since we won't be able to keep this branch synced with master (rebasing would be a nightmare), that's why I set up everything like if it were a separate repo. I think a `npm run backend-script && cd src/frontend && npm run frontend-script` approach might the easiest/safest one (if we don't completely ignore the backend), considering how different the backend and the frontend are from each other.

Reply:
Let's just go for it and fix in follow-ups.

I'm happy for this to get merged if no one else wants to review further.